src/solver/conv/conv_ocl_dir2D_bwdWrW_2.cpp

```diff
@@ -461,6 +461,17 @@ template <int N_BATCH_LOOPS>
 bool ConvOclBwdWrW2<N_BATCH_LOOPS>::IsApplicableBase(const ExecutionContext& ctx,
                                                      const ProblemDescription& problem) const
 {
+
+#if WORKAROUND_SWDEV_503936
+    // Disable this already deprecated solver for gfx11 and gfx12 because it works unstable for
+    // newer video cards
```

Comment:
It's better to place the description above the corresponding `#define WORKAROUND_...`